mw-webapp/src/land/mentorsLandingPage/MentorsLandingPage.tsx

```diff
@@ -77,6 +78,69 @@ export const MentorsLandingPage = observer(() => {
     },
   ];
 
+  const reviewCards: SliderItem[] = [
+    {
+      id: 1,
+      content: (
+        <ReviewCard
+          gradeAmount={5}
+          review={LanguageService.mentorsLanding.reviews.kirillReview.review[language]}
+          reviewerImageUrl="https://drive.google.com/thumbnail?id=1T4XtkEjC7KKgqnox7yE9JttbWdxfzlr4&sz=w1000"
+          reviewerName={LanguageService.mentorsLanding.reviews.kirillReview.mentorName[language]}
+          reviewerProfession={LanguageService.mentorsLanding.reviews.kirillReview.mentorProfession[language]}
+        />
+      ),
+    },
+    {
+      id: 2,
+      content: (
+        <ReviewCard
+          gradeAmount={5}
+          review={LanguageService.mentorsLanding.reviews.viktarReview.review[language]}
+          reviewerImageUrl="https://drive.google.com/thumbnail?id=18oHI9KoiaYvd_UowHyqsJbDLLhmuxPxr&sz=w1000"
+          reviewerName={LanguageService.mentorsLanding.reviews.viktarReview.mentorName[language]}
+          reviewerProfession={LanguageService.mentorsLanding.reviews.viktarReview.mentorProfession[language]}
+        />
+      ),
+    },
+    {
+      id: 3,
+      content: (
+        <ReviewCard
+          gradeAmount={5}
+          review={LanguageService.mentorsLanding.reviews.viktoryiaReview.review[language]}
+          reviewerImageUrl="https://drive.google.com/thumbnail?id=1uEyBTZIon2OFQOOG7pCtkigXYD4YXwc8&sz=w1000"
+          reviewerName={LanguageService.mentorsLanding.reviews.viktoryiaReview.mentorName[language]}
+          reviewerProfession={LanguageService.mentorsLanding.reviews.viktoryiaReview.mentorProfession[language]}
+        />
+      ),
+    },
+    {
+      id: 4,
+      content: (
+        <ReviewCard
+          gradeAmount={4}
+          review={LanguageService.mentorsLanding.reviews.viktoryiaReview.review[language]}
+          reviewerImageUrl="https://drive.google.com/thumbnail?id=1uEyBTZIon2OFQOOG7pCtkigXYD4YXwc8&sz=w1000"
+          reviewerName={LanguageService.mentorsLanding.reviews.viktoryiaReview.mentorName[language]}
+          reviewerProfession={LanguageService.mentorsLanding.reviews.viktoryiaReview.mentorProfession[language]}
+        />
+      ),
+    },
+    {
+      id: 5,
+      content: (
+        <ReviewCard
+          gradeAmount={3}
```

Comment:
let's don't repeat reviews) 3 is enough right now)

Reply:
done